content/docs/join_us/recruitment_FAQs.md

```diff
@@ -19,30 +19,46 @@ Questions are grouped by:
 
 - [Application Stage](#application-stage)
 - [Profile](#profile)
-- [Position](#position)
+- [Role](#role)
 - [Interview](#interview)
 - [Offer](#offer)
 
 ## Application Stage
 
 ### Is the position open to applicants outside of the UK who do not currently have a UK working Visa but could obtain one?
 
-Yes, the positions are open to applicants outside the UK and our HR team ([HR@turing.ac.uk](mailto:HR@turing.ac.uk)) will be able to offer you additional information on the process of obtaining a working Visa.
+Yes, the positions are open to applicants outside the UK and our recruitment team ([recruitment@turing.ac.uk](mailto:recruitment@turing.ac.uk)) will be able to offer you additional information on the process of obtaining a working Visa.
```

Comment:
@IFenton I changed this one as per your comment [here](https://github.com/alan-turing-institute/REG-handbook/pull/157/files#r1765125377)

@cptanalatriste, did you have much contact with the Turing re. applying for your visa? I think recruitment is the best contact for this as they were the ones who sent me the certificate of sponsorship, wondering if your experience was the same :)

Reply:
looking back through my old emails it wasn't recruitment@turing but rather specific people who were emailing me so I've actually just updated it to say speak to the person who's emailing you 😅 i think that's the most appropriate